nodejs/src/access_token_common.js

```diff
@@ -7,11 +7,10 @@ import {ApiCommon} from './api_common.js'
 export class AccessTokenCommon extends ApiCommon {
 
   constructor(api_config, {name = null}) {
-    super();
+    super(api_config);
 
     const auth = api_config.app_id + ':' + api_config.app_secret;
     const b64auth = Buffer.from(auth).toString('base64');
-    this.api_config = api_config;
     this.api_uri = api_config.api_uri;
     this.auth_headers = {'Authorization': 'Basic ' + b64auth}
     if (name) {
```

Comment:
You can use short hand if statement here and a template literal for the string concat. We should probably stick to a convention. Either use "somestring "+somevar; or `somestring ${somevar}`;

this.name = (name) ? name: `access_token_${api_config.version}`;

Reply:
I switched the code to templated strings in as many places as make sense, and used the trinary operator to set access_token.name.